### PR TITLE
Remove language label from the resource state

### PR DESCRIPTION
**Pull request by jcoyne**
Language is now an entity, we only use its identifier

### CODE REVIEW

File: src/Utilities.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import _ from 'lodash'
 import shortid from 'shortid'
 import CryptoJS from 'crypto-js'
 
+export const defaultLangTemplate = 'en'
```
Comment (ndushay) [inline src/Utilities.js]: this var name -- it's not a Template, is it?

File: src/components/editor/property/InputLiteral.jsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ export class InputLiteral extends Component {
     userInputArray.push({
       content: currentcontent,
       id: shortid.generate(),
-      lang: defaultLangTemplate(),
+      lang: defaultLangTemplate,
```
Comment (ndushay) [inline src/components/editor/property/InputLiteral.jsx]: ditto on var name?

File: src/components/editor/property/LanguageButton.jsx [hunk 3]
```diff
@@ -70,7 +70,7 @@ const mapStateToProps = (state, ourProps) => {
   const node = findNode(state.selectorReducer, ourProps.reduxPath)
   const item = node.items.find(item => item.id === ourProps.id)
   return {
-    language: languageLabel(state, item.lang.id),
+    language: languageLabel(state, item.lang),
```
Comment (ndushay) [inline src/components/editor/property/LanguageButton.jsx]: is this function name still appropriate?

Comment (jcoyne) [reply src/components/editor/property/LanguageButton.jsx]: yes, this is looking up the language label from the entity store.

**Comment by jcoyne** [comment]
@ndushay changes made.

**Pull request merged**